packages/mystmd-py/src/mystmd_py/main.py

```diff
@@ -27,9 +27,9 @@ def main():
         raise SystemExit(f"MyST could not determine the version of Node.js: {_version}")
 
     major_version = int(major_version_match[1])
-    if not (major_version in {16, 18} or major_version >= 20):
+    if not (major_version in {18, 20} or major_version >= 22):
```

Comment:
22 is fine?

```suggestion
    if not (major_version in {18, 20, 22} or major_version > 22):
```

Reply:
Yep, just preserving the existing logic!